go.mod

```diff
@@ -83,7 +83,7 @@ require (
 	sigs.k8s.io/release-sdk v0.11.0
 	sigs.k8s.io/release-utils v0.7.7
 	sigs.k8s.io/yaml v1.4.0
-	tags.cncf.io/container-device-interface v0.6.2
+	tags.cncf.io/container-device-interface v0.7.2-0.20240416155956-d7f87365dfce
```

Comment:
You probably want to also downgrade runtime spec.

Reply:
sorry you aren't touching that here.

Reply:
Do we want to wait until there is an official version for cdi?

Reply:
I just wanted to see whether this goes all green without waiting for a new tag. But now we have the new tag, so I update our deps to refer to v0.7.2